galsim/config/value.py

```diff
@@ -428,9 +363,7 @@ def _GetAngleValue(param):
         value = float(value)
         unit = galsim.angle.get_angle_unit(unit)
         return galsim.Angle(value, unit)
-    except KeyboardInterrupt:
-        raise
-    except Exception as e:
+    except (TypeError, AttributeError) as e: # pragma: no cover
```

Comment:
Because I'm catching the other two by explicit type name now, so there is no need to catch and reraise a KeyboardInterrupt.  The main thing is to make sure that KeyboardInterrupts don't get caught and ignored by things like `except Exception as e:` or just `except:`.  

And I think in general, trying to catch the exceptions that you are expecting by name is better than just `except Exception`, so I've been trying to switch to that whenever possible and not too unwieldy.